src/app/signup/components/SignUpAccount.js

```diff
@@ -27,6 +29,14 @@ export function SignUpContainer({ children, sx }) {
 }
 
 export default function SignUpAccount({ goToNextStep }) {
+
+  const mutation = useMutation((value) => signupStep1({ email: value.email, password1: value.password1, password2: value.password2 }));
+
+  async function handleSubmit(values, formikHelpers) {
+    const handle = createHandleSubmit({ mutation, onSuccess: goToNextStep, throwError: true });
```

Comment:
Why throwError true?

Reply:
if we don't need throw error I can delete it